src/config.js

```diff
@@ -21,6 +25,12 @@ const config = convict({
     default: 'development',
     env: 'NODE_ENV'
   },
+  host: {
+    doc: 'The IP address to bind.',
+    format: 'ipaddress',
+    default: '0.0.0.0',
+    env: 'HOST'
+  },
```

Comment:
Binding to `0.0.0.0` as opposed to `localhost` to ensure consistent container behaviour.

Debate whether the convict extension is overkill for this one config value.